Program_Licenses.md

```diff
@@ -38,7 +38,7 @@ The licenses of the open-source software that is contained in these Docker image
 | FLASH | GNU GPLv3 | https://github.com/ebiggers/flash/blob/master/COPYING |
 | Flye | BSD-3 | https://github.com/fenderglass/Flye/blob/flye/LICENSE |
 | Freyja | BSD-2 | https://github.com/andersen-lab/Freyja/blob/main/LICENSE |
-| GAMBIT | GNU aGPLv3 | https://github.com/hesslab-gambit/gambit/blob/master/LICENSE |
+| GAMBIT | GNU aGPLv3 | https://github.com/jlumpe/gambit/blob/master/LICENSE |

```

Comment:
This is going to need more opinions than my own, but I think something like 
```
| GAMBIT | GNU aGPLv3 | https://github.com/jlumpe/gambit/blob/master/LICENSE (formerly located at https://github.com/hesslab-gambit/gambit/blob/master/LICENSE) |
```

would be better. What are your thoughts @kapsakcj, @kevinlibuit, and @k-florek

Reply:
I think it's OK to leave as is. The old link is saved in the commit history here.

When I try to navigate to https://github.com/hesslab-gambit/gambit/blob/master/LICENSE , it redirects me to jlumpe/gambit so I don't think it will be useful to users.

Reply:
That redirect only lasts for about a year. (For example, going to https://github.com/ikkik no longer redirects you to the user https://github.com/erinyoung)

Reply:
I don't think there were too many users who knew or cared about this, to be honest.

Reply:
Staphb's gambit has 1.5K downloads from dockerhub and 599 from quay in the past week...

I think you underestimate yourself.

Reply:
Just commented this above, but

> ditched the old organization and just moved it back under my own user. Aside from this and the BioConda recipe I don't think anyone cared about that URL.

I honestly don't know what fraction of those downloads are from automated builds from BioConda or whatever.